ci/Jenkinsfile

```diff
@@ -35,8 +28,20 @@ pipeline {
       tagFilter: '*',
       type: 'PT_BRANCH'
     )
+    choice(
+      name: 'AGENT',
+      choices: [
+      'linux',
+      'linux-01',
+      'linux-02',
+      'linux-03',
+      'linux-04',
+      'linux-05',
+      ],
+      description: 'Agent name to run tests on it. '
+    )
```

Comment:
What is the point of this?

Reply:
I'm testing why some of the agents don't have a file dialog during test execution
I want to select an agent to execute tests